### ISSUE TITLE
Colors and Themes

### ISSUE THREAD

**Issue by sallar** [andrew-levy/swiftui-react-native#5]
So the `UIColor` object currently in the project, contains only light mode colors. This is fine for light mode, but we have to consider dark mode as well. 

To fix this:

- Instead of storing color values, we can use `DynamicColorIOS` or `PlatformColor` which can handle light/dark variants automatically. 
- In addition, we have to provide a way for the users of the package to overwrite these colors. Either by using a context that they can wrap their app in, or some other means...

What do you think @andrew-levy

**Comment by andrew-levy** [comment]
@sallar I have been working on dark mode for a few days, it is going to be like a ThemeProvider component that will switch between light mode and dark mode colors. I did look at DynamicColorIOS and PlatformColor and they may be useful if the intended users will strictly be ios. I tried running this project on an andrioid device and it actually works pretty well and looks nice, with the exception of a few bugs. But I will keep working on this.
